src/map/status.cpp

```diff
@@ -4297,7 +4297,17 @@ int status_calc_pc_sub(struct map_session_data* sd, enum e_status_calc_opt opt)
 		if (!sd->inventory_data[index])
 			continue;
 
-		base_status->def += sd->inventory_data[index]->def;
+		// Equipments bonus
+		pc_bonus(sd, SP_DEF1, sd->inventory_data[index]->def);
```

Comment:
Correct me if I'm wrong here, but won't these bonuses get removed now on restricted maps? We need to add these bonuses directly to the player so they aren't removed!